pkg/agent/openflow/pipeline.go

```diff
@@ -3106,6 +3107,34 @@ func (f *featureService) gatewaySNATFlows() []binding.Flow {
 	return flows
 }
 
+func (f *featureService) loadBalancerSourceRangesMarkFlows(config *types.ServiceConfig) []binding.Flow {
```

Comment:
We didn't introduce any register mark for LoadBalancer traffic with the check on the source range, and we don't add mark on the packet via the flows built in this function, so maybe remove word "Mark" from the function name? Or rename it as `loadBalancerSourceRangesValidateFlows`?

Reply:
I have changed the designed back because I found the drop action doesn't take effect. For example:

```
table=1, priority=200,ip actions=resubmit(,2),resubmit(,3),resubmit(,4),resubmit(,5)
table=2, priority=0, actions=goto_tables:3
table=3, priority=190,tcp,nw_dst=169.254.169.1,tp_dst=8080 actions=drop
table=4, priority=0,actions=load:0x1->NXM_NX_REG4[19]
table=5, ......
```

In above flows, the packet destined for `169.254.169.1` matched by the flow in table 3 will not be dropped.